00-RELEASENOTES

```diff
@@ -1,16 +1,165 @@
+* Add more filters to CLIENT LIST (#1401)
+* Add `availability_zone` to the HELLO response (#1487)
 
-More information is available at https://valkey.io
+Observability and Monitoring changes
+====================================
+* Extend LATENCY LATEST to add sum / cnt stats (#1570)
+* Add paused_actions for INFO Clients (#1519)
+* Feature COMMANDLOG to record slow execution and large request/reply (#1294)
+* Fix cluster info sent stats for message with light header (#1563)
+* Add latency stats around cluster config file operations (#1534)
 
-Happy hacking!
+Performance/Efficiency Improvements - Core
+==========================================
+* New hash table (#1186)
+* Free strings during BGSAVE/BGAOFRW to reduce copy-on-write (#905)
+* Create an empty lua table with specified initial capacity as much as possible (#1092)
+* Move prepareClientToWrite out of loop for HGETALL command (#1119)
+* Improved hashing algorithm in luaS_newlstr (#1168)
+* Replace dict with new hashtable for sets datatype (#1176)
+* Do security attack check only when command not found to reduce the critical path. (#1212)
+* Accelerate hash table iterator with prefetching (#1501)
+* Replace dict with new hashtable: hash datatype (#1502)
+* Replace dict with new hashtable for sets datatype (#1176)
+* Replace dict with new hashtable: sorted set datatype (#1427)
+* Trim free space from inline command argument strings to avoid excess memory usage (#1213)
+* Increase the IO_THREADS_MAX_NUM. (#1220)
+* Refactor of ActiveDefrag to reduce latencies (#1242)
+* Integrate fast_float to optionally replace strtod (#1260)
+* Improvements for TLS with I/O threads (#1271)
+* Optimize PFCOUNT, PFMERGE command by SIMD acceleration (#1293)
+* Optimize sdscatrepr by batch processing printable characters (#1342)
+* Optimize ZRANK to avoid path comparisons (#1389)
+* Move clientCron onto a separate timer (#1387)
+* Client struct: lazy init components and optimize struct layout (#1405)
+* Offload reading the replication stream to IO threads (#1449)
+* Skip CRC checksumming during diskless full sync with TLS enabled. (#1479)
+* Accelerate hash table iterator with value prefetching (#1568)
+
+New/Modified configurations
+===========================
+* io-threads-do-reads is deprecated and has no effect. (#1138)
+* Import-mode: Avoid expiration and eviction during data syncing (#1185)
+* Relaxed RDB version check (#1604)
+* Deprecate dynamic-hz feature (#1387) 
+* New configuration to control the log format (#1022)
+* Introducing active-defrag-cycle-us for more fine-grinned control of ActiveDefrag run time (#1242) 
+* Introducing new configurations to control the new COMMANDLOG reporting thresholds (#1294)
+
+Build and Packaging changes
+=================================
+* RDMA builtin support (#1209)
+* Fix Valkey binary build workflow, version support changes. (#1429)
+
+Code improvements
+=================
+* Remove Valkey specific changes in jemalloc source code (#1266)
+
```

Comment:
```suggestion
```
I would move this to the performance section. The point wasn't to improve the code, but to make jemalloc more efficient.

Reply:
Also to improve the code by un-patching the dependency. I've seen some distro patch our Makefile to use system jemalloc instead. They didn't get defrag before but now they do. Found it:

https://gitlab.archlinux.org/archlinux/packaging/packages/valkey

Specifically this patch file:

https://gitlab.archlinux.org/archlinux/packaging/packages/valkey/-/blob/main/valkey-5.0-use-system-jemalloc.patch

Reply:
That's a good point, we could also put in the build part. I think we are close, like 90% of the way towards allowing someone to bring their own jemalloc version.